sounds/management/commands/gm_worker_analysis.py

```diff
@@ -0,0 +1,145 @@
+#
```

Comment:
All this funcionatilty will be moved to the celery workers so to the analyzer containers. There will be one worker per type of analyzer, but they should all have common funcionality for soudn conversion (you already added that) and also to do the other things done here like checking for free space.

One difference however is that the new analyzers won't have access to Freesound dababase as they won't have a django installation like these workers. We need to think the best way to create the `SoundAnalysis` objects. I'll discuss that with alastair. Maybe we should add some http endpoint in Freesound that given analysis results will create the `SoundAnalysis` objects.